brainscore/benchmarks/domain_transfer_neural.py

```diff
@@ -0,0 +1,137 @@
+# import general libraries
+import os
+
+# import brain-score specific libraries
+from brainscore.benchmarks._neural_common import NeuralBenchmark, average_repetition
+from brainscore.metrics.ceiling import InternalConsistency
+from brainscore.metrics.regression import CrossRegressedCorrelation, pls_regression, pearsonr_correlation # OODRegressedCorrelation, CrossRegressedCorrelation_with_Background
+from brainscore.utils import LazyLoad
+import brainscore
+from brainio.assemblies import NeuronRecordingAssembly
+
+#### Define constants ####
+
+VISUAL_DEGREES = 8 
+NUMBER_OF_TRIALS = 63 
+BIBTEX = """
+
+"""
+
+ANIMALS = ['Oleo', 'Pico']
+
+CATEGORIES = ['apple', 'bear', 'bird', 'car', 'chair', 'dog', 'elephant', 'face', 'plane', 'zebra']
+SILHOUETTE_DOMAINS = ['convex_hull', 'outline', 'skeleton', 'silhouette']
+
+STYLES = ['hvm', 'convex_hull', 'outline', 'silhouette', 'cartoon', 'line_drawing', 'mosaic', 'painting', 'sketch']
+
+SPLIT_NUMBER = 100
+MAX_NUM_NEURONS = 71
+HVM_TEST_IMAGES_NUM = 30
+OOD_TEST_IMAGES_NUM = 30
+
+
+#### Benchmark implementation ####
+
+def _commonbase(region, identifier_metric_suffix, similarity_metric, ceiler):
+
+    assembly_repetition = LazyLoad(lambda region=region: load_domain_transfer(average_repetitions=False, region=region))
+    assembly = LazyLoad(lambda region=region: load_domain_transfer(average_repetitions=True, region=region))
+    return NeuralBenchmark(identifier=f'dicarlo.Sanghavi2020.domain_transfer.{region}-{identifier_metric_suffix}', version=1,
+                        assembly=assembly, similarity_metric=similarity_metric,
+                        visual_degrees=VISUAL_DEGREES, number_of_trials=NUMBER_OF_TRIALS,
+                        ceiling_func=lambda: ceiler(assembly_repetition),
+                        parent=region,
+                        bibtex=BIBTEX)
+
+
+
+
+def IT_pls():
+    return _commonbase('IT', identifier_metric_suffix='pls',
+                                       similarity_metric=CrossRegressedCorrelation(
+                                           regression=pls_regression(), correlation=pearsonr_correlation(),
+                                           crossvalidation_kwargs = {'stratification_coord': 'object_label',
+                                                                     'preprocess_indices': _preprocess_indices}), 
+                                       ceiler=InternalConsistency())
```

Comment:
can just combine, it's only IT data and we're only using one metric right?
```suggestion
def Igustibagus2024():
    assembly_repetition = LazyLoad(lambda: load_domain_transfer(average_repetitions=False))
    assembly = LazyLoad(lambda: load_domain_transfer(average_repetitions=True))
    similarity_metric=CrossRegressedCorrelation(
                                           regression=ridge_regression(), correlation=pearsonr_correlation(),
                                           crossvalidation_kwargs = {'stratification_coord': 'object_label',
                                                                     'preprocess_indices': _preprocess_indices})
    ceiler=InternalConsistency()
    return NeuralBenchmark(identifier=f'Igustibagus2024-pls', version=1, parent='IT',
                        assembly=assembly, similarity_metric=similarity_metric,
                        visual_degrees=VISUAL_DEGREES, number_of_trials=NUMBER_OF_TRIALS,
                        ceiling_func=lambda: ceiler(assembly_repetition),
                        bibtex=BIBTEX)
```
although maybe we want to do ridge instead of pls?

Reply:
I was thinking about keeping them separated, to make it easy to add new metrics , e.g. ridge as you mentioned

Reply:
let's separate them when we need to ("Premature optimization is the root of all evil")